test/appearance/Montage.rb

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env ruby -w
+
+require 'rmagick'
+require 'test/unit'
+require 'test/unit/ui/console/testrunner'
+require_relative 'appearance_assertion'
+
+class AppearanceMontageUT < Test::Unit::TestCase
+  include AppearanceAssertion
+
+  def test_color
+    imagelist = Magick::ImageList.new(IMAGES_DIR + '/Flower_Hat.jpg')
+
+    new_imagelist = imagelist.montage do
+      self.border_width = 100
+      self.border_color = 'red'
+      self.background_color = 'blue'
+      self.matte_color = 'yellow'
+      self.frame = '10x10'
+      self.gravity = Magick::StaticGravity
```

Comment:
`Magick::StaticGravity` is deprecated. Please change this to `Magick::CenterGravity`.

Reply:
Thanks. Fixed.